sureshassignmentsum.cu

```diff
@@ -0,0 +1,47 @@
+//assignment of arrays sum
+
+#include<iostream>
+using namespace std;
+__global__ void Sum(float* d1_in, float* d2_in, float* d_out, int* d_arraysize)
+{
+    int id = blockIdx.x * blockDim.x + threadIdx.x;
+    if(id < *d_arraysize)
+        d_out[id] = d1_in[id] + d2_in[id];
+}
+int main()
+{
+    int h_arraysize;
+    cout << "Enter the arrays size : ";
+    cin >> h_arraysize;
+    float h1_in[h_arraysize], h_out[h_arraysize] , h2_in[h_arraysize];
+    int Array_Bytes = h_arraysize * sizeof(int);  
+    for(int i=0; i<h_arraysize; i++)
+    {
+        cin >> h1_in[i];
+    }
+    for(int i=0; i<h_arraysize;i++)
+    {
+	cin >> h2_in[i];
+    }
```

Comment:
Hi! this seems fine to check the working of the program but it would be great if you could have a greater number of inputs. The question mentions generating your own input. GPUs are meant for large inputs. Perhaps, consider giving them random inputs or anyhow you wish.